### ISSUE TITLE
Burmese punctuation in Weblate is not supportted or not configured

### ISSUE THREAD

**Issue by WWWesten** [jellyfin/jellyfin#7088]
### Please describe your bug

**Issue**
Burmese punctuation in Weblate is not supportted or not configured. Сolleague @shinkhant started to translate jellyfin-web component to Burmese on Weblate. But there was a problem with checking punctuation. The Burmese writing lacks the usual "European" punctuation (, . : ; ? !) Instead of them, specific Burmese graphemes are used, which Veblate now does not recognize. For example 
source with key DirectPlayHelp (https://translate.jellyfin.org/translate/jellyfin/jellyfin-web/my/?offset=2&q=check%3Aend_stop&sort_by=num_words&checksum=):
`The source file is entirely compatible with this client, and the session is receiving the file without modifications.`
has translation:
`အရင်းအမြစ်ဖိုင်သည် ဤကလိုင်းယင့်နှင့် လုံးဝသဟဇာတဖြစ်ပြီး စက်ရှင်သည် ပြုပြင်မွမ်းမံမှုမရှိဘဲ ဖိုင်ကို လက်ခံရရှိနေပါသည်။`
Last symbol` ။ U+104B MYANMAR SIGN SECTION` is  Burmese "Full Stop". Weblate says "Mismatched full stop 
Source and translation do not both end with a full stop"

**Expected**
Weblate should recognise Burmese punctuation as legal punctuation.

**See also**
https://en.wikipedia.org/wiki/Burmese_alphabet#Punctuation
https://www.1stopasia.com/blog/burmese-script-and-punctuation-everything-you-need-to-know/
https://app.memrise.com/course/437713/myanmar-burmese-alphabetscript/9/

### Jellyfin Version

Other

### if other:

any

### Environment

```markdown
- OS:
- Virtualization:
- Clients:
- Browser:
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by alanazar** [comment]
This can now be closed
Issue has been resolved with the update of Weblate 4.11 and 4.12
https://docs.weblate.org/en/latest/changes.html

**Comment by alanazar** [comment]
This has now been resolved with the latest update to Weblate 4.14.1
Punctuation errors in Jellyfin Weblate have also been corrected & cleared
